src/item.h

```diff
@@ -896,8 +896,6 @@ class item : public JsonSerializer, public JsonDeserializer
  typedef std::vector<item> t_item_vector;
  t_item_vector components;
 
- item clone(bool rand = true);
```

Comment:
>clone wipes out all the details

Just looking at the PR and this jumped out.  Might this boolean have something to do with the rerolling?